### PR TITLE
refactor: changed AboutUs activity into fragment

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes issue #852 

Changes: Added AboutUs fragment.

Screenshots for the change: 

APK for testing: [Compress the *.apk file into a rar or zip file and upload it here]
[app-debug.apk.tar.gz](https://github.com/fossasia/pslab-android/files/1996486/app-debug.apk.tar.gz)

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -653,4 +654,7 @@
     <string name="Search_Bar_Hint_Text">Search..</string>
     <string name="search_menu_title">Search</string>
 
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: What is this one for?

Comment (abhinavraj23) [reply app/src/main/res/values/strings.xml]: I actually didn't coded this manually it was automatically added! Shall I try removing it?

**Comment by abhinavraj23** [comment]
@CloudyPadmal Changes done :+1:

**Comment by abhinavraj23** [comment]
@harsh-2711 I just made one change in the strings.xml file, the rest was done by itself when I gave it a code reformat !!

**Comment by abhinavraj23** [comment]
So I don't think its incorrect as it is automatically done by the system.

**Pull request merged**